src/pages/recipes/menus/index.md

```diff
@@ -0,0 +1,60 @@
+---
+keywords:
+  - UXP Menus
```

Comment:
Menu in InDesign
Menu Items

Remove 'UXP Menus' - We dont have a concept of menus in UXP.